package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "pretest": "eslint .",
-    "test": "jenkins-mocha --recursive"
+    "test": "jenkins-mocha --recursive",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
```

Comment:
Is that a problem with npm publish, or just the environment?

https://docs.npmjs.com/misc/scripts
https://docs.npmjs.com/cli/publish
https://docs.npmjs.com/cli/adduser